akka-docs-dev/rst/java/code/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -55,6 +56,7 @@ public void testHostLevelExample() {
         .via(poolClientFlow)
         .runWith(Sink.<Pair<Try<HttpResponse>, Integer>>head(), materializer);
     //#host-level-example
+    assert(responseFuture != null);
```

Comment:
the intention of that line is not very clear, without a comment (so someone might refactor it away)
suppress warning is just one keyboard click away in eclipse